### PR TITLE
PAE-498 - VUE footer

### PR DESCRIPTION
**Pull request by luismorenolopera**
Description
---------------
This PR adds the footer with the pearson-vue theme.

Screenshots:

image

Ticket link: [PAE-498](https://pearsonadvance.atlassian.net/browse/PAE-498), [PAE-438](https://pearsonadvance.atlassian.net/browse/PAE-438)

Reviewvers
----------------
- @ivanvgh 
- @Squirrel18 
- @diegomillan

### CODE REVIEW

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_footer.scss [hunk 1]
```diff
@@ -1,257 +1,174 @@
...
+
+.apprenticeships-footer {
+   background-color: $app-header-footer !important;
+}
+
+ .wrapper-footer footer#footer-openedx .colophon {
+    float: none;
+    margin: auto;
+    min-height: 30px;
+  }
```
Comment (ivanvgh) [inline edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_footer.scss]: Can we keep the right indentation?

File: edx-platform/pearson-vue-theme/lms/templates/footer.html [hunk 2]
```diff
@@ -92,73 +21,32 @@
...
-              </li>
-            % endfor
-            <li><a href="${footer['edx_org_link']['url']}">${footer['edx_org_link']['text']}</a></li>
+            <li class="nav-legal-01">
+              <a href="https://mylearningx.pearson.com/privacy/">${_('Privacy policy')}</a>
+            </li>
+
+            <li class="nav-legal-02">
+              <a href="https://mylearningx.pearson.com/tos/">${_('Terms of service')}</a>
+            </li>
```
Comment (ivanvgh) [inline edx-platform/pearson-vue-theme/lms/templates/footer.html]: Can we review if those links are working?, [PAE-498](https://pearsonadvance.atlassian.net/browse/PAE-498) expects the following:

Privacy policy: https://mylearningx.pearson.com/privacy
Terms of services: https://mylearningx.pearson.com/tos 

P.D. It works without the slash at the end @luismorenolopera

**Review by ivanvgh — CHANGES_REQUESTED** [review]
Can you please review those comments?

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_footer.scss [hunk 3]
```diff
@@ -1,257 +1,174 @@
...
+}
+
+.wrapper-footer footer#footer-openedx .footer-about-openedx {
+  margin: auto;
+  width: 100%;
+}
+
+div.wrapper-footer footer#footer-openedx .footer-about-openedx a{
+  padding-left:30px;
+  float:left;
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_footer.scss]: Please add a space before the value assignment. @luismorenolopera
`float: left;`

File: edx-platform/pearson-vue-theme/lms/templates/footer.html [hunk 4]
```diff
@@ -10,79 +11,7 @@
...
-              </p>
-            </div>
-          % endif
-        </div>
-      </div>
-    </footer>
-  </div>
-% else:
-  <div class="wrapper wrapper-footer">
+  <div class="wrapper wrapper-footer vue-footer">
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/templates/footer.html:14]: Is it necessary to add the vue-footer class? Why not use the existing classes? @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/templates/footer.html [hunk 5]
```diff
@@ -92,73 +21,32 @@
...
         <nav class="nav-legal" aria-label="${_('Legal')}">
           <ul>
-            % for item_num, link in enumerate(footer['legal_links'], start=1):
-              <li class="nav-legal-0${item_num}">
-                <a href="${link['url']}">${link['title']}</a>
-              </li>
-            % endfor
-            <li><a href="${footer['edx_org_link']['url']}">${footer['edx_org_link']['text']}</a></li>
+            <li class="nav-legal-01">
+              <a href="https://mylearningx.pearson.com/privacy/">${_('Privacy policy')}</a>
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/templates/footer.html]: You don't need to point them to the mylearningx site, point them to the relative path in the theme, which should be:
/privacy & /tos @luismorenolopera

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

File: edx-platform/pearson-vue-theme/lms/templates/footer.html (hunk 5 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/templates/footer.html]: the site is not going to be deployed in mylearninx, but the ticket requires pointing to that site @Squirrel18

File: edx-platform/pearson-vue-theme/lms/templates/footer.html (hunk 4 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/templates/footer.html:14]: I don't think it's necessary, I decided to do it like this to follow the standard of the other themes @Squirrel18

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_footer.scss (hunk 1 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_footer.scss]: done

File: edx-platform/pearson-vue-theme/lms/templates/footer.html (hunk 2 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/templates/footer.html]: done

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_footer.scss (hunk 3 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_footer.scss]: done

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_footer.scss (hunk 1 above)
Comment (ivanvgh) [reply edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_footer.scss]: Can we place the whole 'wrapper-footer' class 1 space indented to the left?, I mean just like 'apprenticeships-footer' indentation @luismorenolopera

**Review by ivanvgh — CHANGES_REQUESTED** [review]

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_footer.scss (hunk 1 above)
Comment (ivanvgh) [reply edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_footer.scss]: Thanks

**Review by ivanvgh — APPROVED** [review]

**Review by Squirrel18 — APPROVED** [review]

**Pull request merged**